org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/ProcessInfoDialog.java

```diff
@@ -216,6 +245,64 @@ public void onEvent(Event event) throws Exception {
 		if (event.getTarget() == btnOk) {
 			this.detach();
 		}
+		if(event.getTarget() == btnPrint) {
+			Clients.showBusy("Processing...");
+			Clients.response(new AuEcho(ProcessInfoDialog.this, "onPrint", null));
+		}
+	}
+	
+	/**
+	 * On Print
+	 */
+	public void onPrint() {
+		// Loop through all items
+		List<File> pdfList = new ArrayList<File>();
+		for (int i = 0; i < m_logs.length; i++)
+		{
+			int RecordID = m_logs[i].getRecord_ID();
+			ReportEngine re = null;
+			
+			re = ReportEngine.get (Env.getCtx(), reportEngineType, RecordID);
+			
+			pdfList.add(re.getPDF());				

```

Comment:
This code block only work if all m_logs record is of same ad_table_id and with record_id > 0. I don't think it is a good idea to assume that's always the case.